src/alto-ui/Datagrid/components/DatagridHeaderCell/DatagridHeaderCellContent.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import ChevronUpIcon from '../../../Icons/ChevronUp';
+import ChevronDownIcon from '../../../Icons/ChevronDown';
+import FilterIcon from '../../../Icons/Filter';
+import VisuallyHidden from '../../../VisuallyHidden';
+import { bemClass } from '../../../helpers/bem';
+
+const DataGridHeaderCellContent = ({ style, sorted, wrapped, context, column }) => {
+  const titleStyle = {
+    ...(wrapped && wrapped !== true ? { maxHeight: `${wrapped * 1.2}em` } : {}),
+  };
+
+  if (!context.onSort || column.sortable === false) {
+    return (
+      <div className={bemClass('DatagridHeaderCell__content', { wrapped })} style={style}>
```

Comment:
```suggestion
      <div className={bemClass('DatagridHeaderCellContent', { wrapped })} style={style}>
```
If you can split scss file also

Reply:
classNames have been changed now in DatagridHeaderCellContent. I also splitted scss file